File: VBLib_Common/Model/MapToQuery.vb
Namespace Model
    Public Class MapToQuery
        Public Property Id As Integer
        Public Property Fluffiness As FluffinessEnum
        Public Property Size As Integer
        Public Overridable Property Club As Club
        Public Overridable Property ClubId As Integer

        Public Overrides Function ToString() As String
            Return _
                $"MapToQuery: Id = {Id}"
        End Function
    End Class
End Namespace

File: FrameworkLib_Common/Model/MapToQuery.cs
namespace FrameworkLib_Common.Model
{
    public class MapToQuery
    {
        public int Id { get; set; }
        public Model.FluffinessEnum Fluffiness { get; set; }
        public int Size { get; set; }
        public virtual Club Club { get; set; } = null!;
        public virtual int ClubId { get; set; }

        public override string ToString()
        {
            return @$"MapToQuery: Id = {Id}";
        }
    }
}